package-lock.json

```diff
@@ -59403,7 +59402,6 @@
         "@wordpress/i18n": "^4.17.0",
         "flagged": "^2.0.6",
         "polished": "^4.2.2",
-        "prop-types": "^15.8.1",
```

Comment:
Why is this being removed here?

Reply:
Because I‘ve gotten rid of prop-types in this package in favor of TS